res/TensorFlowLiteRecipes/ReverseSequence_000/test.recipe

```diff
@@ -0,0 +1,32 @@
+operand {
+  name: "ifm"
+  type: FLOAT32
+  shape { dim: 4 dim: 8 }
+}
+operand {
+  name: "seq_lengths"
+  type: INT32
+  shape { dim: 4 }
+  filler {
+    tag: "explicit"
+    arg: "7" arg: "2" arg: "3" arg: "5" 
+  }
+}
+operand {
+  name: "ofm"
+  type: FLOAT32
+  shape { dim: 4 dim: 8 }
+}
+operation {
+  type: "ReverseSequence"
+  reverse_sequence_options {
+    seq_dim: 0
```

Comment:
ooops, I missed this.
Need to check, possibly it is incorrect value, should be 1

Reply:
```suggestion
    seq_dim: 1
```